ceph_deploy/install.py

```diff
@@ -214,7 +237,10 @@ def uninstall(args):
     for hostname in args.host:
         LOG.debug('Detecting platform for host %s ...', hostname)
 
-        distro = hosts.get(hostname, username=args.username)
+        distro = hosts.get(
+            hostname,
+            username=args.username,
+            use_rhceph=bool(getattr(args, 'use_rhceph', False)))
```

Comment:
I think this can be args.default_release now, right?

the args.use_rhceph was intended to be coming from cephdeploy.conf, which doesn't happen anymore.